src/main/kotlin/io/titandata/titan/providers/local/Commit.kt

```diff
@@ -16,18 +16,34 @@ class Commit (
     private val commitsApi: CommitsApi = CommitsApi(),
     private val uuid: String = UUID.randomUUID().toString().replace("-","")
 ) {
-    fun commit(container: String, message: String) {
+    fun commit(container: String, message: String, tags: List<String>) {
         val repoMetadata = repositoriesApi.getRepository(container).properties
-        val metadata = mapOf(
+        val repoStatus = repositoriesApi.getRepositoryStatus(container)
+        val sourceCommit = repoStatus.sourceCommit
+        val tagMetadata = mutableMapOf<String, String>()
+        for (t in tags) {
```

Comment:
I prefer `for (tag in tags)`. I'm not a fan of single char variable names for the most part. The only exception is `i` for `i--` and `i++`.